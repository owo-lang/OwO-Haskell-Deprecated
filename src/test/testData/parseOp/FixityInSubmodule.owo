module FixityInSubmodule where

  module Submodule where
    infix 1 +

  module Submodule2 where
    infix 1 + - *
    infixl 2 <|> <$> <*>
    infixr 3 <+>
