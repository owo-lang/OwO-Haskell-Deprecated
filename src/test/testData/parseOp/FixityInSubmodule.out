File type: owo
Top module name: .FixityInSubmodule
 submodule .Submodule
 infix 1
  + (69 4 13) (70 4 14)
 submodule .Submodule2
 infixr 3
  <+> (154 9 14) (157 9 17)
 infixl 2
  <|> (129 8 14) (132 8 17)
  <$> (133 8 18) (136 8 21)
  <*> (137 8 22) (140 8 25)
 infix 1
  + (110 7 13) (111 7 14)
  - (112 7 15) (113 7 16)
  * (114 7 17) (115 7 18)
